rseqc-hg38-gencode23/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:14.04
+
+MAINTAINER John Vivian, jtvivian@gmail.com
+
+RUN apt-get update && apt-get install -y \
+            build-essential \
+            python-dev \
+            R-base \
+            samtools \
+            python-pip \
+            python-numpy
+
+RUN pip install cython
+
+# Install and setup RSeQC
+COPY RSeQC-2.6.3.tar.gz /opt/
```

Comment:
I spent about 30 minutes on this before giving up — The tarball is hosted on sourceforge which doesn't provide direct links only redirected mirrors.  `wget` is supposed to follow redirects, but in my tests I had `wget` hang about half the time it attempted to download the file.